src/orange/orangeinp/ConvexRegion.cc

```diff
@@ -382,7 +381,8 @@ void GenTrap::build(ConvexSurfaceBuilder& insert_surface) const
         CELER_ASSERT(SoftZero<real_type>{}(dot_product(d - a, normal)));
 
         // Insert the plane
-        insert_surface(Sense::inside, Plane{normal, offset});
+        auto sense = (offset < 0 ? Sense::outside : Sense::inside);
```

Comment:
Isn't it possible to set up `lo`/`hi` values so that the trapezoid is offset (e.g.) so that it's entirely less than zero, in which case you could have an *inside* sense with a *negative* offset? For example, the face here would have a negative offset, a positive normal, and an *inside* sense:
<img width="428" alt="Screenshot 2024-04-25 at 15 32 04" src="https://github.com/celeritas-project/celeritas/assets/741229/e6527c8e-a179-44f7-8814-ce267e1c224d">

Can you add a test case like that to ensure it gives the expected behavior?